src/components/Status/Status.js

```diff
@@ -14,17 +15,16 @@ class Status extends Component {
 
   render() {
     const { ocupado } = this.state;
-    const { canChange } = this.props;
+    const { canChange } = !!this.props;
```

Comment:
There is no need for this. When the `canChange` prop is not set, it will be undefined and the code will work as intended.

```suggestion
    const { canChange } = this.props;
```